src/main/java/ti4/helpers/PromissoryNoteHelper.java

```diff
@@ -136,6 +138,10 @@ public static List<Button> getPNButtons(Game game, Player player) {
             if (owner == player || pnShortHand.endsWith("_ta")) {
                 continue;
             }
+            if (owner == null) {
+                BotLogger.warning(new LogMessageOrigin(player), pnShortHand + " has no owner in game " + game.getName() + ".");
+                continue;
+            }
```

Comment:
Help with weird situations where PN exists in player's hand, but the owner can't be found.
- example 1: https://discord.com/channels/943410040369479690/1341438476528123904/1376627814006132906
- example 2: https://discord.com/channels/943410040369479690/1341438476528123904/1376262769535483989
- exmaple 3: https://discord.com/channels/943410040369479690/1341438476528123904/1376748910273364080